### DISCUSSION TITLE
Custom contextPath graphQLRoutes

### DISCUSSION THREAD

**Discussion by santanaregis** [ExpediaGroup/graphql-kotlin#1276] [Q&A] [answered]
Hi. I'm with an issue after migrating from version 3 to 4.1.1

Short description: every service has an specific contextPath when deployed to our cloud solution. 
Example: my-app.company.com/**api/v1**/graphql
Property ENV_CONTEXT_PATH = /api/v1
So, before upgrading, I implemented a custom `ContextWebFilter`, filtering every request to ENV_CONTEXT_PATH+`config.endpoint` and ENV_CONTEXT_PATH+`config.subscriptions.endpoint`.  

But, on version 4, I _can't override_ graphQLRoutes()

What's the best solution to this case?
And it's the same to other route configuration, like sdl and playground.

Thanks!

**Comment by smyrick** [comment, marked as answer]
You can set the route path with the configuration: https://opensource.expediagroup.com/graphql-kotlin/docs/server/spring-server/spring-properties
